fe/fe-core/src/test/java/com/starrocks/catalog/ColumnTest.java

```diff
@@ -111,8 +116,104 @@ public void testSerialization() throws Exception {
         file.delete();
     }
 
+    @Test
+    public void testSchemaChangeAllowNormal() throws DdlException {
+        Column oldColumn = new Column("user", ScalarType.createType(PrimitiveType.INT), true, null, false,
+                NULL_DEFAULT_VALUE, "");
+        Column newColumn = new Column("user", ScalarType.createType(PrimitiveType.VARCHAR), true, null, false,
+                NULL_DEFAULT_VALUE, "");
+        oldColumn.checkSchemaChangeAllowed(newColumn);
+
+        oldColumn = new Column("user", ScalarType.createType(PrimitiveType.VARCHAR), true, null, false,
+                new ColumnDef.DefaultValueDef(true, new StringLiteral("0")), "");
+        newColumn = new Column("user1", ScalarType.createType(PrimitiveType.VARCHAR), true, null, false,
```

Comment:
Column name is different.

Reply:
this case is test for change column name.

Reply:
Change column name is not supported currently.